scripts/linters/custom_eslint_checks/lib/rules/break-after-parens.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Lint check to ensure that there is a break after parenthesis
+ * in case of multiline hanging indentation.
+ */
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description: 'There should be a break after parenthesis',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      expectedAfter: 'Expected newline after \'(\'.'
+    }
+  },
+
+  create: function(context) {
+    const sourceCode = context.getSourceCode();
+    const lines = sourceCode.lines;
+
+    return {
+      Program: function checkParenSpaces(node) {
+        const tokens = sourceCode.tokensAndComments;
+        var parensCount = 0;
+        const separators = ['(', '{', '[', ' '];
+        var parens = [];
+        var excluded = false;
+        tokens.forEach((token, i) => {
+          if (token.value === '(' || token.value === ')') {
+            parens.push(token);
+          }
+        });
+        parens.forEach((paren, i) => {
+          const line = lines[paren.loc.start.line - 1].trim();
+          const nextParen = parens[i + 1];
+          if (line.startsWith('it') || line.startsWith('describe') ||
```

Comment:
I would suggest taking `it` block as a function call and it should be fine to do this:

Correct:
```
it('....', function() {

});
```
Incorrect:

```
it('....',
  function(){

});
```

I've left similar comment above*